File: RSPAClient/Core/AtPar.ViewModel/VM_POU_PREF_OPT_BY_PROCEDURE.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_POU_PREF_OPT_BY_PROCEDURE
    {
        public string SPECIALTY_CODE { get; set; }
        public string PROCEDURE_CODE { get; set; }
        public string PROCEDURE_DESCRIPTION { get; set; }
        public Nullable<long> NO_OF_PHYSICIANS { get; set; }
        public Nullable<long> NO_OF_PROCEDURES { get; set; }
        public Nullable<long> NO_OF_PREF_LISTS { get; set; }
        public Nullable<double> EFFICIENCY_PERCENTAGE { get; set; }
        public Nullable<double> TOTAL_PICKED_QTY { get; set; }
        public Nullable<double> TOTAL_PICKED_VALUE { get; set; }
        public Nullable<double> TOTAL_ISSUED_EXISTING_QTY { get; set; }
        public Nullable<double> TOTAL_ISSUED_EXISTING_VALUE { get; set; }
        public Nullable<double> TOTAL_ISSUED_NEW_QTY { get; set; }
        public Nullable<double> TOTAL_ISSUED_NEW_VALUE { get; set; }
        public Nullable<double> TOTAL_RETURN_QTY { get; set; }
        public Nullable<double> TOTAL_RETURN_VALUE { get; set; }
        public Nullable<double> TOTAL_WASTED_QTY { get; set; }
        public Nullable<double> TOTAL_WASTED_VALUE { get; set; }
        public Nullable<double> TOTAL_USAGE { get; set; }
        public List<VM_POU_PREF_OPT_BY_PHYSICIAN> PHYSICIAN_DETAILS { get; set; }        
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_POU_PREF_OPT_BY_PROCEDURE.ts
import { VM_POU_PREF_OPT_BY_PHYSICIAN } from '../Entities/VM_POU_PREF_OPT_BY_PHYSICIAN';

export class VM_POU_PREF_OPT_BY_PROCEDURE {
    public SPECIALTY_CODE: string;
    public PROCEDURE_CODE: string;
    public PROCEDURE_DESCRIPTION: string;
    public NO_OF_PHYSICIANS: any;
    public NO_OF_PROCEDURES: any;
    public NO_OF_PREF_LISTS: any;
    public EFFICIENCY_PERCENTAGE: any;
    public TOTAL_PICKED_QTY: any;
    public TOTAL_PICKED_VALUE: any;
    public TOTAL_ISSUED_EXISTING_QTY: any;
    public TOTAL_ISSUED_EXISTING_VALUE: any;
    public TOTAL_ISSUED_NEW_QTY: any;
    public TOTAL_ISSUED_NEW_VALUE: any;
    public TOTAL_RETURN_QTY: any;
    public TOTAL_RETURN_VALUE: any;
    public TOTAL_WASTED_QTY: any;
    public TOTAL_WASTED_VALUE: any;
    public TOTAL_USAGE: any;
    public ISSHOW: boolean;
    public PHYSICIAN_DETAILS: VM_POU_PREF_OPT_BY_PHYSICIAN[] = [];
    public SHOW_PHYSICIAN_GRID: boolean = false;//ShowPhysicianGrid
    public IS_PREF_CARD_OPT_PHYSICIAN_ROW: boolean=false;
}


      